compiler/z/codegen/OMRCodeGenerator.cpp

```diff
@@ -1474,7 +1474,7 @@ void
 OMR::Z::CodeGenerator::doInstructionSelection()
    {
 
-   _outgoingArgLevelDuringTreeEvaluation = self()->getLinkage()->getNumberOfAllocatedOutgoingArgumentAreas();
+   _outgoingArgLevelDuringTreeEvaluation = 0;
```

Comment:
I believe this field and its accessors can be deleted as well as they are not used anywhere.  Please remove.